phpunit.xml.dist

```diff
@@ -4,4 +4,9 @@
             <directory>./tests</directory>
         </testsuite>
     </testsuites>
+    <filter>
```

Comment:
Sorry I missed to see you have already added this. Was looking at travis and it notifies the problem and ie why ended up adding in #50 .